src/Symfony/Component/Console/Formatter/OutputFormatter.php

```diff
@@ -282,4 +282,27 @@ private function applyCurrentStyle(string $text, string $current, int $width, in
 
         return implode("\n", $lines);
     }
+
+    private function addLineBreaks($string, $charsPerLine)
+    {
+        $result = '';
+
+        if (\function_exists('mb_detect_encoding') && false !== $encoding = mb_detect_encoding($string, null, true)) {
```

Comment:
Because the Console component requires the mbstring Polyfill, we can be certain that `mb_detect_encoding()` is always defined.